CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.3.8 (Next)
```

Comment:
Missed this one, should say 0.4.0 since we bumped major.

Reply:
Rats! Done.